corehq/apps/toggle_ui/views.py

```diff
@@ -157,12 +159,17 @@ def page_context(self):
         return context
 
     def call_save_fn(self, changed_entries, currently_enabled):
-        if self.static_toggle.save_fn is None:
-            return
         for entry in changed_entries:
             if entry.startswith(NAMESPACE_DOMAIN):
                 domain = entry.split(":")[-1]
-                self.static_toggle.save_fn(domain, entry in currently_enabled)
+                if self.static_toggle.save_fn is not None:
+                    self.static_toggle.save_fn(domain, entry in currently_enabled)
+                toggle_js_domain_cachebuster.clear(domain)
+            else:
+                # these are sent down with no namespace
+                assert ':' not in entry
```

Comment:
might want to change this to `assert ':' not in entry, entry` to give more debugging context if this ever fails